### DISCUSSION TITLE
How to set the Post Guidelines/Topic for a Forum Channel?

### DISCUSSION THREAD

**Discussion by Hellinduction** [discord-jda/JDA#2397] [Questions and Help] [answered]
title

**Comment by MinnDevelopment** [comment, marked as answer]
This is called [**topic**](https://ci.dv8tion.net/job/JDA5/javadoc/net/dv8tion/jda/api/entities/channel/concrete/ForumChannel.html#getTopic()) in the API:

- Using `guild.createForumChannel()`: [ChannelAction#setTopic](https://ci.dv8tion.net/job/JDA5/javadoc/net/dv8tion/jda/api/requests/restaction/ChannelAction.html#setTopic(java.lang.String))
    ```java
    guild.createForumChannel(name).setTopic(guidelines).queue();
    ```
- Using `channel.getManager()`: [StandardGuildMessageChannelManager#setTopic](https://ci.dv8tion.net/job/JDA5/javadoc/net/dv8tion/jda/api/managers/channel/middleman/StandardGuildMessageChannelManager.html#setTopic(java.lang.String))
    ```java
    channel.getManager().setTopic(guidelines).queue();
    ```
